AGPs/AGP-XX.md

```diff
@@ -0,0 +1,56 @@
+
+| AGP | Title | Author | Status | Track   | Created |
+|-----|-------|---------------------------|--------|---------|---------|
+| XX  | Nest budget |   louisgrx |      | Finance | 2019-07-10       |
```

Comment:
Is the AGP title "Nest 2019 Budget Extension" or "Nest budget"?

Reply:
Nest 2019 Budget Extension, will correct